docker/staging-compose.yml

```diff
@@ -13,26 +13,32 @@ services:
 
     staging_db:
       build:
-        context: "."
-        dockerfile: "Dockerfile-postgres"
+        context: ".."
+        dockerfile: "docker/Dockerfile-postgres"
       env_file:
-        - .env_staging
+        - ../.env_staging
       hostname: staging_db
+      ports:
+        - "25432:25432"
       volumes:
         - ${STAGING_DATA_DIR}:/var/lib/postgresql/data
       restart: always
+      command: -p 25432
 
     staging_web:
-      build: .
+      build:
+        context: ".."
+        dockerfile: "docker/Dockerfile"
       env_file:
-        - .env_staging
+        - ../.env_staging
       ports:
         - 8080:8000
       environment:
         - MEMCACHED_HOST=staging_memcache
         - CELERY_BROKER_URL=amqp://guest:guest@staging_rabbitmq:5672
         - DATABASE_URL=postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@staging_db:5432/infodengue
```

Comment:
change port to `25432`

`DATABASE_URL=postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@staging_db:25432/infodengue`